README.md

```diff
@@ -64,6 +65,25 @@ io.rewind
 File.write("output.jpeg", io)
 ```
 
+### [StumpyCore](https://github.com/stumpycr/stumpy_core) Conversion
+
+Pluto can convert to and from StumpyCore `Canvas` objects, so any format that Stumpy supports
+can be usable with Pluto as well.
+
+```crystal
+require "pluto"
+require "pluto/stumpy"
+require "stumpy_png"
+
+canvas = StumpyPNG.read("lib/pluto_samples/pluto.png") # => StumpyCore::Canvas
+image = Pluto::ImageRGBA.new(canvas)                   # => Pluto::ImageRGBA
+image.to_stumpy                                        # => StumpyCore::Canvas
+```
+
+**NOTE:** StumpyCore `Canvas` uses a 16-bit color channel whereas Pluto uses 8-bit. Converting from
```

Comment:
IIRC this should render nicely:

```
> **NOTE**: ...
```

Information-wise something like this seems better:

```
Converting from a `StumpyCore::Canvas` created from a 16-bit image will result in a loss of information, since Pluto currently supports only 8 bits.
```